targets/unity-v2/Testing/Tests/Client/ClientApiTests.cs

```diff
@@ -254,14 +257,31 @@ private void GetUserDataCallback2(GetUserDataResult result)
                 int.TryParse(userDataRecord.Value, out actualValue);
             testContext.IntEquals(_testInteger, actualValue);
             testContext.NotNull(userDataRecord, "UserData record not found");
+            testContext.EndTest(UUnitFinishState.PASSED, actualValue.ToString());
+        }
+
+        /// <summary>
+        /// CLIENT API
+        /// Test that the DateTime fields deserialize correctly, and accurately represent UTC time
+        /// Relies on the local machine time and time zone being set correctly.
+        /// Parameter types tested: DateTime
+        /// </summary>
+        [UUnitTest]
+        public void GetServerTime(UUnitTestContext testContext)
+        {
+            clientInstance.GetTime(null, PlayFabUUnitUtils.ApiActionWrapper<GetTimeResult>(testContext, OnGetTimeCallback), PlayFabUUnitUtils.ApiActionWrapper<PlayFabError>(testContext, SharedErrorCallback), testContext);
+        }
+        private void OnGetTimeCallback(GetTimeResult result)
+        {
+            var testContext = (UUnitTestContext)result.CustomData;
 
-            var timeUpdated = userDataRecord.LastUpdated;
+            var serverTime = result.Time;
             var minTest = DateTime.UtcNow - TimeSpan.FromMinutes(5);
             var maxTest = DateTime.UtcNow + TimeSpan.FromMinutes(5);
-            testContext.True(minTest <= timeUpdated && timeUpdated <= maxTest);
+            testContext.True(minTest <= serverTime && serverTime <= maxTest);
 
-            testContext.True(Math.Abs((DateTime.UtcNow - timeUpdated).TotalMinutes) < 5); // Make sure that this timestamp is recent - This must also account for the difference between local machine time and server time
-            testContext.EndTest(UUnitFinishState.PASSED, actualValue.ToString());
+            testContext.True(Math.Abs((DateTime.UtcNow - serverTime).TotalMinutes) < 5); // Make sure that this timestamp is recent - This must also account for the difference between local machine time and server time
```

Comment:
Our documentation for [GetTime](https://api.playfab.com/documentation/Client/method/GetTime) seems to indicate we can have a potential variance of up to 5 seconds. Should this test be in minutes? or Should we expect/test stricter time limits?

Reply:
The Server may have a variance of 5 minutes, but by experimentation I can tell you that cloud instances themselves have upto a ~2 minute variance in time, and that most local machines have the same variance. In fact, there's still a rare but plausible case where 5 minutes isn't even enough, but in practice it seems to be >99% stable.